README.md

```diff
@@ -1 +1,173 @@
-Start of the zendesk README
+# ZenDesk App
+
+## Quick Start
+
+- clone this repo
+- cp `.env.save` `.env`
+  - add your credentials to the `.env` file
+    - ZENDESK_URL - URL to zendesk account
+    - ZENDESK_USERNAME - your zendesk login username
+    - ZENDESK_API_TOKEN - generated via the following steps
+      - zendesk > Admin > CHANNELS > API > Settings
+      - Token Access > Enable
+      - Add API Token
+
+- `npm start` - start the node server
+- `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
+- `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+
+**Current Branches Needed**
+
+The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+
+mm-webapp: `apps-modals`  
+mm-plugin-apps: `apps-modals`
```

Comment:
```suggestion
mattermost-webapp: `feature/cloud-apps`
mattermost-plugin-apps: `master`
```

Reply:
tested and now running with these branches in my development